man/clrtrust.1.md

```diff
@@ -0,0 +1,141 @@
+## SYNOPSIS
+
+**clrtrust** is a tool for generating and managing a centralized trusted
+certificate store.
+
+**clrtrust** \[**-v**|**--verbose**\] \[**-h**|**--help**\] <command> \[options\]
+
+## DESCRIPTION
+
+A trust store contains a set of trusted root certificate authorities (CAs)
+which the operating system and applications should trust. 
+
+The `clrtrust` tool provides a frontend for centralized trust store management.
+It allows viewing, adding (trusting) and removing (distrusting) certificate
+authorities (CAs). It provides some maintenance commands such as re-generating
+the trust store and checking its consistency.
+
+Trusts can be provided by the operating system for out-of-box functionality.
+Trusts can also be provided and modified by privileged users.
+
+## OPTIONS
+
+```
+Usage: clrtrust [-v|--verbose] [-h|--help] <command> [options]
+
+    Commands
+        generate    generates the trust store
+        list        list CAs
+        add         add trust to a CA
+        remove      remove trust to a CA
+        restore     restore trust to previously removed CA
+        check       sanity/consistency check of the trust store
+
+clrtrust <command> --help to get help on specific command.
+```
+
+Commands that modify the store require root privileges.
+
+
+## EXAMPLES
```

Comment:
should be at the bottom?

Reply:
Can you clarify what you meant by this? What should be at the bottom of where?

Reply:
the EXAMPLES section needs to go to the bottom, after you describe all the options and workings.